src/selectors/Entities/vaccinePrescription.js

```diff
@@ -73,3 +73,8 @@ export const selectSelectedVaccinator = state => {
   const { vaccinator } = VaccinePrescriptionState;
   return vaccinator;
 };
+
+export const selectHaveVaccineStock = () =>
+  UIDatabase.objects('Item')
+    .filtered('isVaccine == true')
```

Comment:
perhaps should also have `&& isVisible == true` though I don't know how invisible stock will have available packs. I suppose that is possible